### PR TITLE
i18n package documentation

### PR DESCRIPTION
**Pull request by VictorAvelar**
#### Summary

Adds public documentation for the `@commercetools/i18n` package.

#### Description

Closes SHIELD-443

### CODE REVIEW

File: website/src/content/api-reference/commercetools-frontend-i18n.mdx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+**Using npm**
+
+```sh
+npm install --save @commercetools/i18n
+```
+**Using yarn**
+
+```sh
+yarn add @commercetools/i18n
+```
```
Comment (emmenko) [inline website/src/content/api-reference/commercetools-frontend-i18n.mdx:29]: Can you keep it consistent with the other similar docs pages?

File: website/src/content/api-reference/commercetools-frontend-i18n.mdx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+# Usage
```
Comment (emmenko) [inline website/src/content/api-reference/commercetools-frontend-i18n.mdx:31]: Usually I would like to focus on the page structure first. It's important to define it as it helps identifying the flow of the content and as a consequence where to put what info.

Let's maybe have a sync about it?

**Review by emmenko — COMMENTED** [review]
First contribution, nice! 🥳

File: website/src/content/api-reference/commercetools-frontend-i18n.mdx [hunk 3]
```diff
@@ -0,0 +1,178 @@
+---
+title: "@/i18n"
+---
+<Subtitle>
+
+The internationalization messages from the various application-kit packages are contained as JSON data in this package.
+
+</Subtitle>
+
+# Supported locales
```
[the remark this answers is not in the document]
Comment (VictorAvelar) [reply website/src/content/api-reference/commercetools-frontend-i18n.mdx:10]: I tried to keep the existing structure but it makes sense to keep it consistent.

File: website/src/data/navigation.yaml [hunk 4]
```diff
@@ -103,6 +103,8 @@
     #   path: /api-reference/commercetools-frontend-i18n
```
[the remark this answers is not in the document]
Comment (VictorAvelar) [reply website/src/data/navigation.yaml:103]: Will do, completely miss them. 👍🏽

**Comment by emmenko** [comment]
Superseeded by https://github.com/commercetools/commercetools-docs/pull/5211

**Pull request closed without merging**